lib/merge.ts

```diff
@@ -448,6 +516,9 @@ export function normalizeSourceFile(f: ts.SourceFile, fc: FacadeConverter, expli
         continue;
       }
       const name = base.ident(clause.types[0].expression);
+      if (!classes.has(name)) {
```

Comment:
document why this hack is needed. if we ever support exporting ts files that depend on other ts files this could be a problem.